tests/e2e/specs/api/DevfileAcceptanceTestAPI.spec.ts

```diff
@@ -3,47 +3,102 @@ import { DevWorkspaceConfigurationHelper } from '../../utils/DevWorkspaceConfigu
+
+            test('Collect build commands from the devfile', async function (): Promise<void> {
+                if (devfileContext.devfile.commands === undefined) {
+                    Logger.info(`Devfile does not contains any commands.`);
+                } else {
+                    devfileContext.devfile.commands.forEach((command: any) => {
+                        if (command.exec.group.kind === 'build') {
+                            Logger.debug(`Build command found: ${command.exec.commandLine}`);
+                            devfilesBuildCommands.push(command);
+                        }
+                    });
+                }
+            });
 
-        kubernetesCommandLineToolsExecutor = new KubernetesCommandLineToolsExecutor(devWorkspaceName);
-        kubernetesCommandLineToolsExecutor.loginToOcp();
-    });
+            test('Create DevWorkspace', async function (): Promise<void> {
+                const devWorkspaceConfigurationYamlString: string = await devWorkspaceConfigurationHelper.getDevWorkspaceConfigurationYamlAsString(devfileContext);
+                const applyOutput: ShellString = kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsStringOutput(devWorkspaceConfigurationYamlString);
 
-    test('Create DevWorkspace', async function (): Promise<void> {
-        const devWorkspaceConfigurationYamlString: string = await devWorkspaceConfigurationHelper.getDevWorkspaceConfigurationYamlAsString(devfileContext);
-        const applyOutput: ShellString = kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsStringOutput(devWorkspaceConfigurationYamlString);
+                expect(applyOutput.stdout)
+                    .contains('devworkspacetemplate')
+                    .and.contains('devworkspace')
+                    .and.contains.oneOf(['created', 'configured']);
 
-        expect(applyOutput.stdout)
-            .contains('devworkspacetemplate')
-            .and.contains('devworkspace')
-            .and.contains.oneOf(['created', 'configured']);
+            });
 
-    });
+            test('Wait until DevWorkspace has status "ready"', async function (): Promise<void> {
+                expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
+            });
 
-    test('Wait until DevWorkspace has status "ready"', async function (): Promise<void> {
-        expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
-    });
+            test('Check if project was created', function (): void {
+                clonedProjectName = StringUtil.getProjectNameFromGitUrl(devfileSample.link);
+                expect(containerTerminal.ls().stdout).includes(clonedProjectName);
+            });
 
-    suiteTeardown('Delete DevWorkspace', async function (): Promise<void> {
-        kubernetesCommandLineToolsExecutor.deleteDevWorkspace();
-    });
-});
+            test('Check if files were imported', function (): void {
```

Comment:
IMHO, it's more clear to say `Check if project is imported`

Reply:
But we have almost the same name of the test above. It is the same in factories and quarkus ui. Maybe "Check if project files were imported"?